pkg/kubelet/kuberuntime/kuberuntime_container_linux_test.go

```diff
@@ -674,3 +675,115 @@ func TestGenerateLinuxContainerConfigSwap(t *testing.T) {
 		})
 	}
 }
+
+func TestGenerateLinuxContainerResources(t *testing.T) {
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.InPlacePodVerticalScaling, true)()
```

Comment:
We need test cases to verify behaviour with both feature gate on and off.

Reply:
fixed. will be in next commit.

Reply:
Added test for featuregate values of both true and false.
https://github.com/kubernetes/kubernetes/pull/102884/files#diff-95938bea0cf5278aa810cf7b839aa50d3d08879b3bb430e3c35bf06d349885c9R717